### ISSUE TITLE
Unable to play some videos

### ISSUE THREAD

**Issue by CuloArdido** [iv-org/invidious#602]
Since I discovered Invidious a few months ago I've been using it as a replacement to YouTube every time I can but there are some videos that do not work, why is this? Next I leave the videos that have given me problems:

| video URL                      | `allowed_regions` ([API](https://github.com/omarroth/invidious/wiki/API))                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          | Blacklisted regions                                                                                                                                  | Whitelisted regions                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
|--------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| invidio.us/watch?v=Yc5sgGMmGUQ | AD, AE, AF, AG, AI, AL, AM, AO, AQ, AR, AS, AT, AU, AW, AX, AZ, BA, BB, BD, BE, BF, BG, BH, BI, BJ, BL, BM, BN, BO, BQ, BR, BS, BT, BV, BW, BY, BZ, CA, CC, CD, CF, CG, CH, CI, CK, CL, CM, CN, CO, CR, CU, CV, CW, CX, CY, CZ, DE, DJ, DK, DM, DO, DZ, EC, EE, EG, EH, ER, ES, ET, FI, FJ, FK, FM, FO, FR, GA, GB, GD, GE, GF, GG, GH, GI, GL, GM, GN, GP, GQ, GR, GS, GT, GU, GW, GY, HK, HM, HN, HR, HT, HU, ID, IE, IL, IM, IN, IO, IQ, IR, IS, IT, JE, JM, JO, JP, KE, KG, KH, KI, KM, KN, KP, KR, KW, KY, KZ, LA, LB, LC, LI, LK, LR, LS, LT, LU, LV, LY, MA, MC, MD, ME, MF, MG, MH, MK, ML, MM, MN, MO, MP, MQ, MR, MS, MT, MU, MV, MW, MX, MY, MZ, NA, NC, NE, NF, NG, NI, NL, NO, NP, NR, NU, NZ, OM, PA, PE, PF, PG, PH, PK, PL, PM, PN, PS, PT, PW, PY, QA, RE, RO, RS, RU, RW, SA, SB, SC, SD, SE, SG, SH, SI, SJ, SK, SL, SM, SN, SO, SR, SS, ST, SV, SX, SY, SZ, TC, TD, TF, TG, TH, TJ, TK, TL, TM, TN, TO, TR, TT, TV, TW, TZ, UA, UG, UM, US, UY, UZ, VA, VC, VE, VG, VI, VN, VU, WF, WS, YE, YT, ZA, ZM, ZW     | PR                                                                                                                                                   |                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
| invidio.us/watch?v=INS2Dsm1_Ew | AG, AI, AR, AW, BB, BO, BQ, BS, BZ, CL, CO, CR, CU, CW, DM, DO, EC, GD, GF, GP, GT, GY, HN, HT, JM, KN, KY, LC, MQ, MS, MX, NI, PA, PE, PY, SR, SV, TC, TT, UY, VC, VE, VG                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |                                                                                                                                                      | AG, AI, AR, AW, BB, BO, BQ, BS, BZ, CL, CO, CR, CU, CW, DM, DO, EC, GD, GF, GP, GT, GY, HN, HT, JM, KN, KY, LC, MQ, MS, MX, NI, PA, PE, PY, SR, SV, TC, TT, UY, VC, VE, VG                                                                                                                                                                                                                                                                                                 |
| invidio.us/watch?v=h4bX2boNO5g | AG, AI, AR, AW, BB, BO, BQ, BS, BZ, CL, CO, CR, CU, CW, DM, DO, EC, GD, GF, GP, GT, GY, HN, HT, JM, KN, KY, LC, MQ, MS, MX, NI, PA, PE, PY, SR, SV, TC, TT, UY, VC, VE, VG                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |                                                                                                                                                      | AG, AI, AR, AW, BB, BO, BQ, BS, BZ, CL, CO, CR, CU, CW, DM, DO, EC, GD, GF, GP, GT, GY, HN, HT, JM, KN, KY, LC, MQ, MS, MX, NI, PA, PE, PY, SR, SV, TC, TT, UY, VC, VE, VG                                                                                                                                                                                                                                                                                                 |
| invidio.us/watch?v=iOSUriAFLgk | AD, AF, AG, AI, AL, AM, AO, AQ, AR, AS, AT, AU, AW, AX, AZ, BA, BB, BD, BE, BF, BG, BI, BJ, BL, BM, BN, BO, BQ, BR, BS, BT, BV, BW, BY, BZ, CA, CC, CD, CF, CG, CH, CI, CK, CL, CM, CN, CO, CR, CU, CV, CW, CX, CY, CZ, DE, DJ, DK, DM, DO, EC, EE, EH, ER, ES, ET, FI, FJ, FK, FM, FO, FR, GA, GB, GD, GE, GF, GG, GH, GI, GL, GM, GN, GP, GQ, GR, GS, GT, GU, GW, GY, HK, HM, HN, HR, HT, HU, ID, IE, IL, IM, IN, IO, IS, IT, JE, JM, JP, KE, KG, KH, KI, KM, KN, KP, KR, KY, KZ, LA, LC, LI, LK, LR, LS, LT, LU, LV, MC, MD, ME, MF, MG, MH, MK, ML, MM, MN, MO, MP, MQ, MR, MS, MT, MU, MV, MW, MX, MY, MZ, NA, NC, NE, NF, NG, NI, NL, NO, NP, NR, NU, NZ, PA, PE, PF, PG, PH, PK, PL, PM, PN, PR, PT, PW, PY, RE, RO, RS, RU, RW, SB, SC, SD, SE, SG, SH, SI, SJ, SK, SL, SM, SN, SO, SR, SS, ST, SV, SX, SZ, TC, TD, TF, TG, TH, TJ, TK, TL, TM, TO, TR, TT, TV, TW, TZ, UA, UG, UM, US, UY, UZ, VA, VC, VE, VG, VI, VN, VU, WF, WS, YT, ZA, ZM, ZW                                                                         | AE, BH, DZ, EG, IQ, IR, JO, KW, LB, LY, MA, OM, PS, QA, SA, SY, TN, YE                                                                               |                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
| invidio.us/watch?v=vuhHw2VeGCY | AG, AI, AQ, AR, AS, AU, AW, BB, BD, BL, BM, BN, BO, BQ, BR, BS, BT, BV, BZ, CA, CC, CK, CL, CN, CO, CR, CU, CW, CX, DM, DO, EC, ER, FJ, FK, FM, GD, GF, GP, GS, GT, GU, GY, HK, HM, HN, HT, ID, IN, IO, JM, JP, KH, KI, KN, KP, KR, KY, LA, LC, LK, MF, MH, MM, MN, MO, MP, MQ, MS, MV, MX, MY, NC, NF, NI, NP, NR, NU, NZ, PA, PE, PF, PG, PH, PK, PN, PR, PW, PY, SB, SG, SJ, SR, SV, SX, TC, TF, TH, TK, TL, TO, TT, TV, TW, UM, UY, VA, VC, VE, VG, VI, VN, VU, WF, WS                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |                                                                                                                                                      | AG, AI, AQ, AR, AS, AU, AW, BB, BD, BL, BM, BN, BO, BQ, BR, BS, BT, BV, BZ, CA, CC, CK, CL, CN, CO, CR, CU, CW, CX, DM, DO, EC, ER, FJ, FK, FM, GD, GF, GP, GS, GT, GU, GY, HK, HM, HN, HT, ID, IN, IO, JM, JP, KH, KI, KN, KP, KR, KY, LA, LC, LK, MF, MH, MM, MN, MO, MP, MQ, MS, MV, MX, MY, NC, NF, NI, NP, NR, NU, NZ, PA, PE, PF, PG, PH, PK, PN, PR, PW, PY, SB, SG, SJ, SR, SV, SX, TC, TF, TH, TK, TL, TO, TT, TV, TW, UM, UY, VA, VC, VE, VG, VI, VN, VU, WF, WS |
| invidio.us/watch?v=d8Qd4K8Medc | AD, AE, AF, AG, AI, AL, AM, AO, AQ, AR, AS, AT, AU, AW, AX, AZ, BA, BB, BD, BE, BF, BG, BH, BI, BJ, BL, BM, BN, BO, BQ, BR, BS, BT, BV, BW, BY, BZ, CA, CC, CD, CF, CG, CH, CI, CK, CL, CM, CN, CO, CR, CU, CV, CW, CX, CY, CZ, DE, DJ, DK, DM, DO, DZ, EC, EE, EG, EH, ER, ES, ET, FI, FJ, FK, FM, FO, FR, GA, GB, GD, GE, GF, GG, GH, GI, GL, GM, GN, GP, GQ, GR, GS, GT, GU, GW, GY, HK, HM, HN, HR, HT, HU, ID, IE, IL, IM, IN, IO, IQ, IR, IS, IT, JE, JM, JO, JP, KE, KG, KH, KI, KM, KN, KP, KR, KW, KY, KZ, LA, LB, LC, LI, LK, LR, LS, LT, LU, LV, LY, MA, MC, MD, ME, MF, MG, MH, MK, ML, MM, MN, MO, MP, MQ, MR, MS, MT, MU, MV, MW, MX, MY, MZ, NA, NC, NE, NF, NG, NI, NL, NO, NP, NR, NU, NZ, OM, PA, PE, PF, PG, PH, PK, PL, PM, PN, PS, PT, PW, PY, QA, RE, RO, RS, RU, RW, SA, SB, SC, SD, SE, SG, SH, SI, SJ, SK, SL, SM, SN, SO, SR, SS, ST, SV, SX, SY, SZ, TC, TD, TF, TG, TH, TJ, TK, TL, TM, TN, TO, TR, TT, TV, TW, TZ, UA, UG, UM, US, UY, UZ, VA, VC, VE, VG, VI, VN, VU, WF, WS, YE, YT, ZA, ZM, ZW     | PR                                                                                                                                                   |                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
| invidio.us/watch?v=FxhJLCRsz98 | AD, AE, AF, AG, AI, AL, AM, AO, AQ, AR, AS, AT, AU, AW, AX, AZ, BA, BB, BD, BE, BF, BG, BH, BI, BJ, BL, BM, BN, BO, BQ, BR, BS, BT, BV, BW, BY, BZ, CA, CC, CD, CF, CG, CH, CI, CK, CL, CM, CN, CO, CR, CU, CV, CW, CX, CY, CZ, DE, DJ, DK, DM, DO, DZ, EC, EE, EG, EH, ER, ES, ET, FI, FJ, FK, FM, FO, FR, GA, GB, GD, GE, GF, GG, GH, GI, GL, GM, GN, GP, GQ, GR, GS, GT, GU, GW, GY, HK, HM, HN, HR, HT, HU, ID, IE, IL, IM, IN, IO, IQ, IR, IS, IT, JE, JM, JO, JP, KE, KG, KH, KI, KM, KN, KP, KR, KW, KY, KZ, LA, LB, LC, LI, LK, LR, LS, LT, LU, LV, LY, MA, MC, MD, ME, MF, MG, MH, MK, ML, MM, MN, MO, MP, MQ, MR, MS, MT, MU, MV, MW, MX, MY, MZ, NA, NC, NE, NF, NG, NI, NL, NO, NP, NR, NU, NZ, OM, PA, PE, PF, PG, PH, PK, PL, PM, PN, PS, PT, PW, PY, QA, RE, RO, RS, RU, RW, SA, SB, SC, SD, SE, SG, SH, SI, SJ, SK, SL, SM, SN, SO, SR, SS, ST, SV, SX, SY, SZ, TC, TD, TF, TG, TH, TJ, TK, TL, TM, TN, TO, TR, TT, TV, TW, TZ, UA, UG, UM, US, UY, UZ, VA, VC, VE, VG, VI, VN, VU, WF, WS, YE, YT, ZA, ZM, ZW     | PR                                                                                                                                                   |                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
| invidio.us/watch?v=I1OOIgcEADk | AD, AE, AF, AG, AI, AL, AM, AO, AQ, AR, AS, AT, AU, AW, AX, AZ, BA, BB, BD, BE, BF, BG, BH, BI, BJ, BL, BM, BN, BO, BQ, BR, BS, BT, BV, BW, BY, BZ, CA, CC, CD, CF, CG, CH, CI, CK, CL, CM, CN, CO, CR, CU, CV, CW, CX, CY, CZ, DE, DJ, DK, DM, DO, DZ, EC, EE, EG, EH, ER, ES, ET, FI, FJ, FK, FM, FO, FR, GA, GB, GD, GE, GF, GG, GH, GI, GL, GM, GN, GP, GQ, GR, GS, GT, GU, GW, GY, HK, HM, HN, HR, HT, HU, ID, IE, IL, IM, IN, IO, IQ, IR, IS, IT, JE, JM, JO, KE, KG, KH, KI, KM, KN, KP, KR, KW, KY, KZ, LA, LB, LC, LI, LK, LR, LS, LT, LU, LV, LY, MA, MC, MD, ME, MF, MG, MH, MK, ML, MM, MN, MO, MP, MQ, MR, MS, MT, MU, MV, MW, MX, MY, MZ, NA, NC, NE, NF, NG, NI, NL, NO, NP, NR, NU, NZ, OM, PA, PE, PF, PG, PH, PK, PL, PM, PN, PR, PS, PT, PW, PY, QA, RE, RO, RS, RU, RW, SA, SB, SC, SD, SE, SG, SH, SI, SJ, SK, SL, SM, SN, SO, SR, SS, ST, SV, SX, SY, SZ, TC, TD, TF, TG, TH, TJ, TK, TL, TM, TN, TO, TR, TT, TV, TW, TZ, UA, UG, UM, US, UY, UZ, VA, VC, VE, VG, VI, VN, VU, WF, WS, YE, YT, ZA, ZM, ZW     | JP                                                                                                                                                   |                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
| invidio.us/watch?v=0fo-8LkzC9U | AD, AE, AF, AG, AI, AL, AM, AO, AQ, AR, AS, AT, AU, AW, AX, AZ, BA, BB, BD, BE, BF, BG, BH, BI, BJ, BL, BM, BN, BO, BQ, BR, BS, BT, BV, BW, BY, BZ, CA, CC, CD, CF, CG, CH, CI, CK, CL, CM, CN, CO, CR, CU, CV, CW, CX, CY, CZ, DE, DJ, DK, DM, DO, DZ, EC, EE, EG, EH, ER, ES, ET, FI, FJ, FK, FM, FO, FR, GA, GB, GD, GE, GF, GG, GH, GI, GL, GM, GN, GP, GQ, GR, GS, GT, GU, GW, GY, HK, HM, HN, HR, HT, HU, ID, IE, IL, IM, IN, IO, IQ, IR, IS, IT, JE, JM, JO, KE, KG, KH, KI, KM, KN, KP, KR, KW, KY, KZ, LA, LB, LC, LI, LK, LR, LS, LT, LU, LV, LY, MA, MC, MD, ME, MF, MG, MH, MK, ML, MM, MN, MO, MP, MQ, MR, MS, MT, MU, MV, MW, MX, MY, MZ, NA, NC, NE, NF, NG, NI, NL, NO, NP, NR, NU, NZ, OM, PA, PE, PF, PG, PH, PK, PL, PM, PN, PR, PS, PT, PW, PY, QA, RE, RO, RS, RU, RW, SA, SB, SC, SD, SE, SG, SH, SI, SJ, SK, SL, SM, SN, SO, SR, SS, ST, SV, SX, SY, SZ, TC, TD, TF, TG, TH, TJ, TK, TL, TM, TN, TO, TR, TT, TV, TW, TZ, UA, UG, UM, US, UY, UZ, VA, VC, VE, VG, VI, VN, VU, WF, WS, YE, YT, ZA, ZM, ZW     | JP                                                                                                                                                   |                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
| invidio.us/watch?v=5Mcj_YtQhbQ | AR, AT, AU, BA, BE, BG, BO, BR, BY, CH, CL, CO, CR, CY, CZ, DE, DK, DO, EC, EE, ES, FI, FR, GR, GT, HN, HR, HU, ID, IN, IS, IT, JP, KR, LI, LT, LU, LV, MK, MT, MX, NI, NL, NO, NZ, PA, PE, PL, PT, PY, RO, RS, RU, SE, SI, SK, SV, TR, UA, US, UY, VE, ZA                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |                                                                                                                                                      | AR, AT, AU, BA, BE, BG, BO, BR, BY, CH, CL, CO, CR, CY, CZ, DE, DK, DO, EC, EE, ES, FI, FR, GR, GT, HN, HR, HU, ID, IN, IS, IT, JP, KR, LI, LT, LU, LV, MK, MT, MX, NI, NL, NO, NZ, PA, PE, PL, PT, PY, RO, RS, RU, SE, SI, SK, SV, TR, UA, US, UY, VE, ZA                                                                                                                                                                                                                 |
| invidio.us/watch?v=5-3KI7up4eM | AD, AF, AG, AI, AL, AM, AO, AQ, AR, AS, AT, AU, AW, AX, AZ, BA, BB, BD, BE, BF, BG, BI, BJ, BL, BM, BN, BO, BQ, BR, BS, BT, BV, BW, BY, BZ, CA, CC, CD, CF, CG, CH, CI, CK, CL, CM, CN, CO, CR, CU, CV, CW, CX, CY, CZ, DE, DJ, DK, DM, DO, EC, EE, EH, ER, ES, ET, FI, FJ, FK, FM, FO, FR, GA, GB, GD, GE, GF, GG, GH, GI, GL, GM, GN, GP, GQ, GR, GS, GT, GU, GW, GY, HK, HM, HN, HR, HT, HU, ID, IE, IL, IM, IN, IO, IS, IT, JE, JM, JP, KE, KG, KH, KI, KM, KN, KP, KR, KY, KZ, LA, LC, LI, LK, LR, LS, LT, LU, LV, MC, MD, ME, MF, MG, MH, MK, ML, MM, MN, MO, MP, MQ, MR, MS, MT, MU, MV, MW, MX, MY, MZ, NA, NC, NE, NF, NG, NI, NL, NO, NP, NR, NU, NZ, PA, PE, PF, PG, PH, PK, PL, PM, PN, PR, PT, PW, PY, RE, RO, RS, RU, RW, SB, SC, SD, SE, SG, SH, SI, SJ, SK, SL, SM, SN, SO, SR, SS, ST, SV, SX, SZ, TC, TD, TF, TG, TH, TJ, TK, TL, TM, TO, TR, TT, TV, TW, TZ, UA, UG, UM, US, UY, UZ, VA, VC, VE, VG, VI, VN, VU, WF, WS, YT, ZA, ZM, ZW                                                                         | AE, BH, DZ, EG, IQ, IR, JO, KW, LB, LY, MA, OM, PS, QA, SA, SY, TN, YE                                                                               |                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
| invidio.us/watch?v=kMML6qvVT54 | AD, AE, AF, AG, AI, AL, AM, AO, AQ, AR, AS, AT, AU, AW, AX, AZ, BA, BB, BD, BE, BF, BG, BH, BI, BJ, BL, BM, BN, BO, BQ, BR, BS, BT, BV, BW, BY, BZ, CA, CC, CD, CF, CG, CH, CI, CK, CL, CM, CN, CO, CR, CU, CV, CW, CX, CY, CZ, DE, DJ, DK, DM, DO, DZ, EC, EE, EG, EH, ER, ES, ET, FI, FJ, FK, FM, FO, FR, GA, GB, GD, GE, GF, GG, GH, GI, GL, GM, GN, GP, GQ, GR, GS, GT, GU, GW, GY, HK, HM, HN, HR, HT, HU, ID, IE, IL, IM, IN, IO, IQ, IR, IS, IT, JE, JM, JO, JP, KE, KG, KH, KI, KM, KN, KP, KR, KW, KY, KZ, LA, LB, LC, LI, LK, LR, LS, LT, LU, LV, LY, MA, MC, MD, ME, MF, MG, MH, MK, ML, MM, MN, MO, MP, MQ, MR, MS, MT, MU, MV, MW, MX, MY, MZ, NA, NC, NE, NF, NG, NI, NL, NO, NP, NR, NU, NZ, OM, PA, PE, PF, PG, PH, PK, PL, PM, PN, PR, PS, PT, PW, PY, QA, RE, RO, RS, RU, RW, SA, SB, SC, SD, SE, SG, SH, SI, SJ, SK, SL, SM, SN, SO, SR, SS, ST, SV, SX, SY, SZ, TC, TD, TF, TG, TH, TJ, TK, TL, TM, TN, TO, TR, TT, TV, TW, TZ, UA, UG, UM, US, UY, UZ, VA, VC, VE, VG, VI, VN, VU, WF, WS, YE, YT, ZA, ZM, ZW |                                                                                                                                                      |                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
| invidio.us/watch?v=ccq8ctJ4PKY | AD, AE, AF, AG, AI, AL, AM, AO, AQ, AR, AS, AT, AU, AW, AX, AZ, BA, BB, BD, BE, BF, BG, BH, BI, BJ, BL, BM, BN, BO, BQ, BR, BS, BT, BV, BW, BY, BZ, CA, CC, CD, CF, CG, CH, CI, CK, CL, CM, CN, CO, CR, CU, CV, CW, CX, CY, CZ, DE, DJ, DK, DM, DO, DZ, EC, EE, EG, EH, ER, ES, ET, FI, FJ, FK, FM, FO, FR, GA, GB, GD, GE, GF, GG, GH, GI, GL, GM, GN, GP, GQ, GR, GS, GT, GU, GW, GY, HK, HM, HN, HR, HT, HU, ID, IE, IL, IM, IN, IO, IQ, IR, IS, IT, JE, JM, JO, JP, KE, KG, KH, KI, KM, KN, KP, KR, KW, KY, KZ, LA, LB, LC, LI, LK, LR, LS, LT, LU, LV, LY, MA, MC, MD, ME, MF, MG, MH, MK, ML, MM, MN, MO, MP, MQ, MR, MS, MT, MU, MV, MW, MX, MY, MZ, NA, NC, NE, NF, NG, NI, NL, NO, NP, NR, NU, NZ, OM, PA, PE, PF, PG, PH, PK, PL, PM, PN, PR, PS, PT, PW, PY, QA, RE, RO, RS, RU, RW, SA, SB, SC, SD, SE, SG, SH, SI, SJ, SK, SL, SM, SN, SO, SR, SS, ST, SV, SX, SY, SZ, TC, TD, TF, TG, TH, TJ, TK, TL, TM, TN, TO, TR, TT, TV, TW, TZ, UA, UG, UM, UY, UZ, VA, VC, VE, VG, VI, VN, VU, WF, WS, YE, YT, ZA, ZM, ZW     | US                                                                                                                                                   |                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
| invidio.us/watch?v=SI4Wf_qse5k | AD, AE, AF, AG, AI, AL, AM, AO, AQ, AS, AT, AU, AW, AX, AZ, BA, BB, BD, BE, BF, BG, BH, BI, BJ, BL, BM, BN, BO, BQ, BR, BS, BT, BV, BW, BY, BZ, CA, CC, CD, CF, CG, CH, CI, CK, CL, CM, CN, CO, CR, CU, CV, CW, CX, CY, CZ, DE, DJ, DK, DM, DO, DZ, EC, EE, EG, EH, ER, ES, ET, FI, FJ, FK, FM, FO, FR, GA, GB, GD, GE, GF, GG, GH, GI, GL, GM, GN, GP, GQ, GR, GS, GT, GU, GW, GY, HK, HM, HN, HR, HT, HU, ID, IE, IL, IM, IN, IO, IQ, IR, IS, IT, JE, JM, JO, JP, KE, KG, KH, KI, KM, KN, KP, KR, KW, KY, KZ, LA, LB, LC, LI, LK, LR, LS, LT, LU, LV, LY, MA, MC, MD, ME, MF, MG, MH, MK, ML, MM, MN, MO, MP, MQ, MR, MS, MT, MU, MV, MW, MX, MY, MZ, NA, NC, NE, NF, NG, NI, NL, NO, NP, NR, NU, NZ, OM, PA, PE, PF, PG, PH, PK, PL, PM, PN, PR, PS, PT, PW, PY, QA, RE, RO, RS, RU, RW, SA, SB, SC, SD, SE, SG, SH, SI, SJ, SK, SL, SM, SN, SO, SR, SS, ST, SV, SX, SY, SZ, TC, TD, TF, TG, TH, TJ, TK, TL, TM, TN, TO, TR, TT, TV, TW, TZ, UA, UG, UM, US, UY, UZ, VA, VC, VE, VG, VI, VN, VU, WF, WS, YE, YT, ZA, ZM, ZW     | AR                                                                                                                                                   |                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
| invidio.us/watch?v=dHVBxb_sg_k | AD, AE, AG, AI, AL, AM, AO, AQ, AR, AS, AT, AU, AW, AX, AZ, BA, BB, BE, BF, BG, BI, BJ, BL, BM, BO, BQ, BR, BS, BV, BW, BY, BZ, CA, CC, CD, CF, CG, CH, CI, CK, CL, CM, CO, CR, CU, CV, CW, CX, CY, CZ, DE, DJ, DK, DM, DO, DZ, EC, EE, EG, EH, ER, ES, ET, FI, FJ, FK, FM, FO, FR, GA, GB, GD, GE, GF, GG, GH, GI, GL, GM, GN, GP, GQ, GR, GS, GT, GU, GW, GY, HM, HN, HR, HT, HU, IE, IL, IM, IN, IO, IS, IT, JE, JM, JP, KE, KI, KM, KN, KP, KR, KY, LC, LI, LR, LS, LT, LU, LV, LY, MA, MC, MD, ME, MF, MG, MH, MK, ML, MO, MP, MQ, MR, MS, MT, MU, MW, MX, MZ, NA, NC, NE, NF, NG, NI, NL, NO, NR, NU, NZ, PA, PE, PF, PG, PL, PM, PN, PR, PS, PT, PW, PY, RE, RO, RS, RU, RW, SB, SC, SD, SE, SH, SI, SJ, SK, SL, SM, SN, SO, SR, SS, ST, SV, SX, SZ, TC, TD, TF, TG, TK, TL, TN, TO, TR, TT, TV, TZ, UA, UG, UM, US, UY, VA, VC, VE, VG, VI, VU, WF, WS, YT, ZA, ZM, ZW                                                                                                                                                     | AF, BD, BH, BN, BT, CN, HK, ID, IQ, IR, JO, KG, KH, KW, KZ, LA, LB, LK, MM, MN, MV, MY, NP, OM, PH, PK, QA, SA, SG, SY, TH, TJ, TM, TW, UZ, VN, YE |                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
| invidio.us/watch?v=WXToe26mwTo | AD, AE, AF, AG, AI, AL, AM, AO, AQ, AR, AS, AT, AU, AW, AX, AZ, BA, BB, BD, BE, BF, BG, BH, BI, BJ, BL, BM, BN, BO, BQ, BR, BS, BT, BV, BW, BY, BZ, CA, CC, CD, CF, CG, CH, CI, CK, CL, CM, CN, CO, CR, CU, CV, CW, CX, CY, CZ, DE, DJ, DK, DM, DO, DZ, EC, EE, EG, EH, ER, ES, ET, FI, FJ, FK, FM, FO, FR, GA, GB, GD, GE, GF, GG, GH, GI, GL, GM, GN, GP, GQ, GR, GS, GT, GU, GW, GY, HK, HM, HN, HR, HT, HU, ID, IE, IL, IM, IN, IO, IQ, IR, IS, IT, JE, JM, JO, JP, KE, KG, KH, KI, KM, KN, KP, KR, KW, KY, KZ, LA, LB, LC, LI, LK, LR, LS, LT, LU, LV, LY, MA, MC, MD, ME, MF, MG, MH, MK, ML, MM, MN, MO, MP, MQ, MR, MS, MT, MU, MV, MW, MX, MY, MZ, NA, NC, NE, NF, NG, NI, NL, NO, NP, NR, NU, NZ, OM, PA, PE, PF, PG, PH, PK, PL, PM, PN, PS, PT, PW, PY, QA, RE, RO, RS, RU, RW, SA, SB, SC, SD, SE, SG, SH, SI, SJ, SK, SL, SM, SN, SO, SR, SS, ST, SV, SX, SY, SZ, TC, TD, TF, TG, TH, TJ, TK, TL, TM, TN, TO, TR, TT, TV, TW, TZ, UA, UG, UM, US, UY, UZ, VA, VC, VE, VG, VI, VN, VU, WF, WS, YE, YT, ZA, ZM, ZW     | PR                                                                                                                                                   |                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
| invidio.us/watch?v=wmmEvPqN4Zk | AD, AE, AF, AG, AI, AL, AM, AO, AQ, AR, AS, AT, AU, AW, AX, AZ, BA, BB, BD, BE, BF, BG, BH, BI, BJ, BL, BM, BN, BO, BQ, BR, BS, BT, BV, BW, BY, BZ, CA, CC, CD, CF, CG, CH, CI, CK, CL, CM, CN, CO, CR, CU, CV, CW, CX, CY, CZ, DE, DJ, DK, DM, DO, DZ, EC, EE, EG, EH, ER, ES, ET, FI, FJ, FK, FM, FO, FR, GA, GB, GD, GE, GF, GG, GH, GI, GL, GM, GN, GP, GQ, GR, GS, GT, GU, GW, GY, HK, HM, HN, HR, HT, HU, ID, IE, IL, IM, IN, IO, IQ, IR, IS, IT, JE, JM, JO, JP, KE, KG, KH, KI, KM, KN, KP, KR, KW, KY, KZ, LA, LB, LC, LI, LK, LR, LS, LT, LU, LV, LY, MA, MC, MD, ME, MF, MG, MH, MK, ML, MM, MN, MO, MP, MQ, MR, MS, MT, MU, MV, MW, MX, MY, MZ, NA, NC, NE, NF, NG, NI, NL, NO, NP, NR, NU, NZ, OM, PA, PE, PF, PG, PH, PK, PL, PM, PN, PR, PS, PT, PW, PY, QA, RE, RO, RS, RU, RW, SA, SB, SC, SD, SE, SG, SH, SI, SJ, SK, SL, SM, SN, SO, SR, SS, ST, SV, SX, SY, SZ, TC, TD, TF, TG, TH, TJ, TK, TL, TM, TN, TO, TR, TT, TV, TW, TZ, UA, UG, UM, UY, UZ, VA, VC, VE, VG, VI, VN, VU, WF, WS, YE, YT, ZA, ZM, ZW     | US                                                                                                                                                   |                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
| invidio.us/watch?v=KMVfkku-B8o | AD, AE, AF, AG, AI, AL, AM, AO, AQ, AR, AS, AT, AU, AW, AX, AZ, BA, BB, BD, BE, BF, BG, BH, BI, BJ, BL, BM, BN, BO, BQ, BR, BS, BT, BV, BW, BY, BZ, CA, CC, CD, CF, CG, CH, CI, CK, CL, CM, CN, CO, CR, CU, CV, CW, CX, CY, CZ, DE, DJ, DK, DM, DO, DZ, EC, EE, EG, EH, ER, ES, ET, FI, FJ, FK, FM, FO, FR, GA, GB, GD, GE, GF, GG, GH, GI, GL, GM, GN, GP, GQ, GR, GS, GT, GU, GW, GY, HK, HM, HN, HR, HT, HU, ID, IE, IL, IM, IN, IO, IQ, IR, IS, IT, JE, JM, JO, JP, KE, KG, KH, KI, KM, KN, KP, KR, KW, KY, KZ, LA, LB, LC, LI, LK, LR, LS, LT, LU, LV, LY, MA, MC, MD, ME, MF, MG, MH, MK, ML, MM, MN, MO, MP, MQ, MR, MS, MT, MU, MV, MW, MX, MY, MZ, NA, NC, NE, NF, NG, NI, NL, NO, NP, NR, NU, NZ, OM, PA, PE, PF, PG, PH, PK, PL, PM, PN, PR, PS, PT, PW, PY, QA, RE, RO, RS, RU, RW, SA, SB, SC, SD, SE, SG, SH, SI, SJ, SK, SL, SM, SN, SO, SR, SS, ST, SV, SX, SY, SZ, TC, TD, TF, TG, TH, TJ, TK, TL, TM, TN, TO, TR, TT, TV, TW, TZ, UA, UG, UM, UY, UZ, VA, VC, VE, VG, VI, VN, VU, WF, WS, YE, YT, ZA, ZM, ZW     | US                                                                                                                                                   |                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
| invidio.us/watch?v=c26IXiQo5w0 | AD, AE, AF, AG, AI, AL, AM, AO, AQ, AR, AS, AT, AU, AW, AX, AZ, BA, BB, BD, BE, BF, BG, BH, BI, BJ, BL, BM, BN, BO, BQ, BR, BS, BT, BV, BW, BY, BZ, CA, CC, CD, CF, CG, CH, CI, CK, CL, CM, CN, CO, CR, CU, CV, CW, CX, CY, CZ, DE, DJ, DK, DM, DO, DZ, EC, EE, EG, EH, ER, ES, ET, FI, FJ, FK, FM, FO, FR, GA, GB, GD, GE, GF, GG, GH, GI, GL, GM, GN, GP, GQ, GR, GS, GT, GU, GW, GY, HK, HM, HN, HR, HT, HU, ID, IE, IL, IM, IN, IO, IQ, IR, IS, IT, JE, JM, JO, JP, KE, KG, KH, KI, KM, KN, KP, KR, KW, KY, KZ, LA, LB, LC, LI, LK, LR, LS, LT, LU, LV, LY, MA, MC, MD, ME, MF, MG, MH, MK, ML, MM, MN, MO, MP, MQ, MR, MS, MT, MU, MV, MW, MX, MY, MZ, NA, NC, NE, NF, NG, NI, NL, NO, NP, NR, NU, NZ, OM, PA, PE, PF, PG, PH, PK, PL, PM, PN, PR, PS, PT, PW, PY, QA, RE, RO, RS, RU, RW, SA, SB, SC, SD, SE, SG, SH, SI, SJ, SK, SL, SM, SN, SO, SR, SS, ST, SV, SX, SY, SZ, TC, TD, TF, TG, TH, TJ, TK, TL, TM, TN, TO, TR, TT, TV, TW, TZ, UA, UG, UM, UY, UZ, VA, VC, VE, VG, VI, VN, VU, WF, WS, YE, YT, ZA, ZM, ZW     | US                                                                                                                                                   |                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
| invidio.us/watch?v=cpzL0H6mkQc | AD, AE, AF, AG, AI, AL, AM, AO, AQ, AR, AS, AT, AU, AW, AX, AZ, BA, BB, BD, BE, BF, BG, BH, BI, BJ, BL, BM, BN, BO, BQ, BR, BS, BT, BV, BW, BY, BZ, CA, CC, CD, CF, CG, CH, CI, CK, CL, CM, CN, CO, CR, CU, CV, CW, CX, CY, CZ, DE, DJ, DK, DM, DO, DZ, EC, EE, EG, EH, ER, ES, ET, FI, FJ, FK, FM, FO, FR, GA, GB, GD, GE, GF, GG, GH, GI, GL, GM, GN, GP, GQ, GR, GS, GT, GU, GW, GY, HK, HM, HN, HR, HT, HU, ID, IE, IL, IM, IN, IO, IQ, IR, IS, IT, JE, JM, JO, JP, KE, KG, KH, KI, KM, KN, KP, KR, KW, KY, KZ, LA, LB, LC, LI, LK, LR, LS, LT, LU, LV, LY, MA, MC, MD, ME, MF, MG, MH, MK, ML, MM, MN, MO, MP, MQ, MR, MS, MT, MU, MV, MW, MX, MY, MZ, NA, NC, NE, NF, NG, NI, NL, NO, NP, NR, NU, NZ, OM, PA, PE, PF, PG, PH, PK, PL, PM, PN, PR, PS, PT, PW, PY, QA, RE, RO, RS, RU, RW, SA, SB, SC, SD, SE, SG, SH, SI, SJ, SK, SL, SM, SN, SO, SR, SS, ST, SV, SX, SY, SZ, TC, TD, TF, TG, TH, TJ, TK, TL, TM, TN, TO, TR, TT, TV, TW, TZ, UA, UG, UM, UY, UZ, VA, VC, VE, VG, VI, VN, VU, WF, WS, YE, YT, ZA, ZM, ZW     | US                                                                                                                                                   |                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
| invidio.us/watch?v=mJqS3R8dfFM | AD, AE, AF, AG, AI, AL, AM, AO, AQ, AR, AS, AT, AU, AW, AX, AZ, BA, BB, BD, BE, BF, BG, BH, BI, BJ, BL, BM, BN, BO, BQ, BR, BS, BT, BV, BW, BY, BZ, CA, CC, CD, CF, CG, CH, CI, CK, CL, CM, CN, CO, CR, CU, CV, CW, CX, CY, CZ, DE, DJ, DK, DM, DO, DZ, EC, EE, EG, EH, ER, ES, ET, FI, FJ, FK, FM, FO, FR, GA, GB, GD, GE, GF, GG, GH, GI, GL, GM, GN, GP, GQ, GR, GS, GT, GU, GW, GY, HK, HM, HN, HR, HT, HU, ID, IE, IL, IM, IN, IO, IQ, IR, IS, IT, JE, JM, JO, JP, KE, KG, KH, KI, KM, KN, KP, KR, KW, KY, KZ, LA, LB, LC, LI, LK, LR, LS, LT, LU, LV, LY, MA, MC, MD, ME, MF, MG, MH, MK, ML, MM, MN, MO, MP, MQ, MR, MS, MT, MU, MV, MW, MX, MY, MZ, NA, NC, NE, NF, NG, NI, NL, NO, NP, NR, NU, NZ, OM, PA, PE, PF, PG, PH, PK, PL, PM, PN, PR, PS, PT, PW, PY, QA, RE, RO, RS, RU, RW, SA, SB, SC, SD, SE, SG, SH, SI, SJ, SK, SL, SM, SN, SO, SR, SS, ST, SV, SX, SY, SZ, TC, TD, TF, TG, TH, TJ, TK, TL, TM, TN, TO, TR, TT, TV, TW, TZ, UA, UG, UM, UY, UZ, VA, VC, VE, VG, VI, VN, VU, WF, WS, YE, YT, ZA, ZM, ZW     | US                                                                                                                                                   |                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
| invidio.us/watch?v=_xMucDVHag8 | AD, AE, AF, AG, AI, AL, AM, AO, AQ, AR, AS, AT, AU, AW, AX, AZ, BA, BB, BD, BE, BF, BG, BH, BI, BJ, BL, BM, BN, BO, BQ, BR, BS, BT, BV, BW, BY, BZ, CA, CC, CD, CF, CG, CH, CI, CK, CL, CM, CN, CO, CR, CU, CV, CW, CX, CY, CZ, DE, DJ, DK, DM, DO, DZ, EC, EE, EG, EH, ER, ES, ET, FI, FJ, FK, FM, FO, FR, GA, GB, GD, GE, GF, GG, GH, GI, GL, GM, GN, GP, GQ, GR, GS, GT, GU, GW, GY, HK, HM, HN, HR, HT, HU, ID, IE, IL, IM, IN, IO, IQ, IR, IS, IT, JE, JM, JO, JP, KE, KG, KH, KI, KM, KN, KP, KR, KW, KY, KZ, LA, LB, LC, LI, LK, LR, LS, LT, LU, LV, LY, MA, MC, MD, ME, MF, MG, MH, MK, ML, MM, MN, MO, MP, MQ, MR, MS, MT, MU, MV, MW, MX, MY, MZ, NA, NC, NE, NF, NG, NI, NL, NO, NP, NR, NU, NZ, OM, PA, PE, PF, PG, PH, PK, PL, PM, PN, PR, PS, PT, PW, PY, QA, RE, RO, RS, RU, RW, SA, SB, SC, SD, SE, SG, SH, SI, SJ, SK, SL, SM, SN, SO, SR, SS, ST, SV, SX, SY, SZ, TC, TD, TF, TG, TH, TJ, TK, TL, TM, TN, TO, TR, TT, TV, TW, TZ, UA, UG, UM, UY, UZ, VA, VC, VE, VG, VI, VN, VU, WF, WS, YE, YT, ZA, ZM, ZW     | US                                                                                                                                                   |                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
| invidio.us/watch?v=Q3FBADNgPS0 | AD, AE, AF, AG, AI, AL, AM, AO, AQ, AR, AS, AT, AU, AW, AX, AZ, BA, BB, BD, BE, BF, BG, BH, BI, BJ, BL, BM, BN, BO, BQ, BR, BS, BT, BV, BW, BY, BZ, CA, CC, CD, CF, CG, CH, CI, CK, CL, CM, CN, CO, CR, CU, CV, CW, CX, CY, CZ, DE, DJ, DK, DM, DO, DZ, EC, EE, EG, EH, ER, ES, ET, FI, FJ, FK, FM, FO, FR, GA, GB, GD, GE, GF, GG, GH, GI, GL, GM, GN, GP, GQ, GR, GS, GT, GU, GW, GY, HK, HM, HN, HR, HT, HU, ID, IE, IL, IM, IN, IO, IQ, IR, IS, IT, JE, JM, JO, JP, KE, KG, KH, KI, KM, KN, KP, KR, KW, KY, KZ, LA, LB, LC, LI, LK, LR, LS, LT, LU, LV, LY, MA, MC, MD, ME, MF, MG, MH, MK, ML, MM, MN, MO, MP, MQ, MR, MS, MT, MU, MV, MW, MX, MY, MZ, NA, NC, NE, NF, NG, NI, NL, NO, NP, NR, NU, NZ, OM, PA, PE, PF, PG, PH, PK, PL, PM, PN, PR, PS, PT, PW, PY, QA, RE, RO, RS, RU, RW, SA, SB, SC, SD, SE, SG, SH, SI, SJ, SK, SL, SM, SN, SO, SR, SS, ST, SV, SX, SY, SZ, TC, TD, TF, TG, TH, TJ, TK, TL, TM, TN, TO, TR, TT, TV, TW, TZ, UA, UG, UM, UY, UZ, VA, VC, VE, VG, VI, VN, VU, WF, WS, YE, YT, ZA, ZM, ZW     | US                                                                                                                                                   |                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
| invidio.us/watch?v=yYo1jsM-KhA | AD, AE, AF, AG, AI, AL, AM, AO, AQ, AR, AS, AT, AU, AW, AX, AZ, BA, BB, BD, BE, BF, BG, BH, BI, BJ, BL, BM, BN, BO, BQ, BR, BS, BT, BV, BW, BY, BZ, CC, CD, CF, CG, CH, CI, CK, CL, CM, CN, CO, CR, CU, CV, CW, CX, CY, CZ, DE, DJ, DK, DM, DO, DZ, EC, EE, EG, EH, ER, ES, ET, FI, FJ, FK, FM, FO, FR, GA, GB, GD, GE, GF, GG, GH, GI, GL, GM, GN, GP, GQ, GR, GS, GT, GU, GW, GY, HK, HM, HN, HR, HT, HU, ID, IE, IL, IM, IN, IO, IQ, IR, IS, IT, JE, JM, JO, JP, KE, KG, KH, KI, KM, KN, KP, KR, KW, KY, KZ, LA, LB, LC, LI, LK, LR, LS, LT, LU, LV, LY, MA, MC, MD, ME, MF, MG, MH, MK, ML, MM, MN, MO, MP, MQ, MR, MS, MT, MU, MV, MW, MX, MY, MZ, NA, NC, NE, NF, NG, NI, NL, NO, NP, NR, NU, NZ, OM, PA, PE, PF, PG, PH, PK, PL, PM, PN, PR, PS, PT, PW, PY, QA, RE, RO, RS, RU, RW, SA, SB, SC, SD, SE, SG, SH, SI, SJ, SK, SL, SM, SN, SO, SR, SS, ST, SV, SX, SY, SZ, TC, TD, TF, TG, TH, TJ, TK, TL, TM, TN, TO, TR, TT, TV, TW, TZ, UA, UG, UM, UY, UZ, VA, VC, VE, VG, VI, VN, VU, WF, WS, YE, YT, ZA, ZM, ZW | CA, US | |
| invidio.us/watch?v=s1S--jFuly4 | AD, AE, AF, AG, AI, AL, AM, AO, AQ, AR, AS, AT, AW, AX, AZ, BA, BB, BD, BE, BF, BG, BH, BI, BJ, BL, BM, BN, BO, BQ, BR, BS, BT, BV, BW, BY, BZ, CA, CD, CF, CG, CH, CI, CL, CM, CN, CO, CR, CU, CV, CW, CY, CZ, DE, DJ, DK, DM, DO, DZ, EC, EE, EG, EH, ER, ES, ET, FI, FJ, FK, FM, FO, FR, GA, GB, GD, GE, GF, GG, GH, GI, GL, GM, GN, GP, GQ, GR, GS, GT, GU, GW, GY, HK, HN, HR, HT, HU, ID, IE, IL, IM, IN, IO, IQ, IR, IS, IT, JE, JM, JO, JP, KE, KG, KH, KM, KN, KP, KR, KW, KY, KZ, LA, LB, LC, LI, LK, LR, LS, LT, LU, LV, LY, MA, MC, MD, ME, MF, MG, MH, MK, ML, MM, MN, MO, MP, MQ, MR, MS, MT, MU, MV, MW, MX, MY, MZ, NA, NC, NE, NG, NI, NL, NO, NP, NU, OM, PA, PE, PF, PG, PH, PK, PL, PM, PN, PR, PS, PT, PW, PY, QA, RE, RO, RS, RU, RW, SA, SC, SD, SE, SG, SH, SI, SJ, SK, SL, SM, SN, SO, SR, SS, ST, SV, SX, SY, SZ, TC, TD, TF, TG, TH, TJ, TK, TL, TM, TN, TR, TT, TW, TZ, UA, UG, UM, US, UY, UZ, VA, VC, VE, VG, VI, VN, WF, YE, YT, ZA, ZM, ZW | AU, CC, CK, CX, HM, KI, NF, NR, NZ, SB, TO, TV, VU, WS | | 
| invidio.us/watch?v=vuK1U76dRNE | AD, AE, AF, AG, AI, AL, AM, AO, AQ, AR, AS, AU, AW, AX, AZ, BA, BB, BD, BE, BF, BG, BH, BI, BJ, BL, BM, BN, BO, BQ, BR, BS, BT, BV, BW, BY, BZ, CA, CC, CD, CF, CG, CH, CI, CK, CL, CM, CN, CO, CR, CU, CV, CW, CX, CY, CZ, DJ, DK, DM, DO, DZ, EC, EE, EG, EH, ER, ES, ET, FI, FJ, FK, FM, FO, FR, GA, GB, GD, GE, GF, GG, GH, GI, GL, GM, GN, GP, GQ, GR, GS, GT, GU, GW, GY, HK, HM, HN, HR, HT, HU, ID, IE, IL, IM, IN, IO, IQ, IR, IS, IT, JE, JM, JO, JP, KE, KG, KH, KI, KM, KN, KP, KR, KW, KY, KZ, LA, LB, LC, LK, LR, LS, LT, LU, LV, LY, MA, MC, MD, ME, MF, MG, MH, MK, ML, MM, MN, MO, MP, MQ, MR, MS, MT, MU, MV, MW, MX, MY, MZ, NA, NC, NE, NF, NG, NI, NL, NO, NP, NR, NU, NZ, OM, PA, PE, PF, PG, PH, PK, PL, PM, PN, PR, PS, PT, PW, PY, QA, RE, RO, RS, RU, RW, SA, SB, SC, SD, SE, SG, SH, SI, SJ, SK, SL, SM, SN, SO, SR, SS, ST, SV, SX, SY, SZ, TC, TD, TF, TG, TH, TJ, TK, TL, TM, TN, TO, TR, TT, TV, TW, TZ, UA, UG, UM, US, UY, UZ, VA, VC, VE, VG, VI, VN, VU, WF, WS, YE, YT, ZA, ZM, ZW | AT, DE, LI | |
| invidio.us/watch?v=P7l6iQe0x2Q | AD, AE, AF, AG, AI, AL, AM, AO, AQ, AR, AS, AT, AU, AW, AX, AZ, BA, BB, BD, BE, BF, BG, BH, BI, BJ, BL, BM, BN, BO, BQ, BR, BS, BT, BV, BW, BY, BZ, CA, CC, CD, CF, CG, CH, CI, CK, CL, CM, CN, CO, CR, CU, CV, CW, CX, CY, CZ, DE, DJ, DK, DM, DO, DZ, EC, EE, EG, EH, ER, ES, ET, FI, FJ, FK, FM, FO, FR, GA, GB, GD, GE, GF, GG, GH, GI, GL, GM, GN, GP, GQ, GR, GS, GT, GU, GW, GY, HM, HN, HR, HT, HU, ID, IE, IL, IM, IN, IO, IQ, IR, IS, IT, JE, JM, JO, JP, KE, KG, KH, KI, KM, KN, KP, KR, KW, KY, KZ, LA, LB, LC, LI, LK, LR, LS, LT, LU, LV, LY, MA, MC, MD, ME, MF, MG, MH, MK, ML, MM, MN, MP, MQ, MR, MS, MT, MU, MV, MW, MX, MY, MZ, NA, NC, NE, NF, NG, NI, NL, NO, NP, NR, NU, NZ, OM, PA, PE, PF, PG, PH, PK, PL, PM, PN, PR, PS, PT, PW, PY, QA, RE, RO, RS, RU, RW, SA, SB, SC, SD, SE, SG, SH, SI, SJ, SK, SL, SM, SN, SO, SR, SS, ST, SV, SX, SY, SZ, TC, TD, TF, TG, TH, TJ, TK, TL, TM, TN, TO, TR, TT, TV, TW, TZ, UA, UG, UM, US, UY, UZ, VA, VC, VE, VG, VI, VN, VU, WF, WS, YE, YT, ZA, ZM, ZW | HK, MO | |

I would have put them with the full URL but the last time that I added many links in an issue my account was blocked by spam.
When I open any of those videos I get the error "The media could not be loaded, either because the server or network failed or because the format is not supported."
I checked them in three different browsers to discard software problems or lack of codecs.
If I discover more videos with the same problem I will add it to the list.

**Comment by Perflyst** [comment]
Please add &local=true at the end and it should work

**Comment by CuloArdido** [comment]
I've been doing some tests with the videos and indeed all can be played by adding local=true in the link (although as a third of them I could see after several attempts) but the strangest thing is that all can be seen on YouTube (except one that is effectively blocked where I live) but not in Invidious, so I am not sure that the problem is only due to a geographical limitation (I converted the list into a table to better reflect this). I would like someone else to review the videos that are not blocked in their respective countries to confirm if it only happens to me or if it is something generalized.
Even if it were the case, if the user tries to see some blocked video in his country maybe it could show him some message like this:
```
[Optionally insert warning sign here]
<h2>Video blocked geographically</h2>
This video was blocked in your country by X because of Y, but you can still try to watch it from <a href="https://invidio.us/watch?v=xxxxxxxxxxxx&local=true">here</a>.
```
On the other hand, when the video is only blocked in the country where the Invidious instance is located (a few must be included in the table), the ideal would be to load it transparently to the user through a proxy or redirect it automatically to the page with local=true.
I imagine that all this could be done by comparing the ips through something like [GeoIP](https://dev.maxmind.com/geoip/geoip2/geolite2/).

**Comment by omarroth** [comment]
> all can be seen on YouTube (except one that is effectively blocked where I live) but not in Invidious

For some videos, YouTube blocks access from regions other than where the video was originally requested, even if the video is not region blocked, see [this comment](https://github.com/omarroth/invidious/issues/524#issuecomment-499144951).

Several of the videos you tested are also blocked in the US, which is where Invidious is hosted. I expect it may have taken a couple tries for Invidious to get a working proxy. You may try using [an instance](https://github.com/omarroth/invidious/wiki/Invidious-Instances) in your region to see if you encounter fewer issues with playback.

> ideal would be to load it transparently to the user through a proxy or redirect it automatically to the page with local=true

This is something Invidious already does. You might take a look at [this comment](https://github.com/omarroth/invidious/issues/453#issuecomment-477574597) for some more explanation of how videos are proxied.

**Comment by CuloArdido** [comment]
>For some videos, YouTube blocks access from regions other than where the video was originally requested, even if the video is not region blocked

I understand that videos blocked in my country or in the United States cannot be viewed directly, but what about those that are accessible in both regions? Why can't they be viewed unless a proxy is used? (I ask for curiosity, I imagine there will be a technical reason behind that) For example, according to the API these videos are accessible from both locations but still cannot be viewed from Invidious:
invidio.us/watch?v=Yc5sgGMmGUQ
invidio.us/watch?v=iOSUriAFLgk
invidio.us/watch?v=d8Qd4K8Medc
invidio.us/watch?v=FxhJLCRsz98
invidio.us/watch?v=I1OOIgcEADk
invidio.us/watch?v=0fo-8LkzC9U
invidio.us/watch?v=5Mcj_YtQhbQ
invidio.us/watch?v=5-3KI7up4eM
invidio.us/watch?v=kMML6qvVT54
invidio.us/watch?v=dHVBxb_sg_k
invidio.us/watch?v=WXToe26mwTo
>I expect it may have taken a couple tries for Invidious to get a working proxy.

Maybe it could be improved with something like [ProxyCheck](https://github.com/AlexeyFreelancer/ProxyChecker) but testing which has the most stable streaming.
>This is something Invidious already does.

But not in the way I described it. What I had explained before is rather for the users who do not know about this and they find a message as generic and unclear as "The media could not be loaded ..." when try to play a video, which may think that Invidious works badly or something like that and end up opening issues like this, so I suggested that instead could show a more explanatory message like the one I proposed in my previous comment (assuming that it can unequivocally detect that the video can not be loaded due to a geographical block and not for other reasons) and in case the country where Invidious is running is blocked, then proxify the video automatically without having to add local=true in the url.
Something simpler to implement could be a message like this that always shows in the bottom of the video:
`Problems to load? Try <a href="https://invidio.us/watch?v=xxxxxxxxxxxx&local=true">this</a>.`

**Comment by fliegentoeter** [comment]
Is there a way I can get invidious to add "&local=true" to every URL by default? This fixes almost every unplayable video for me. But it's a pain to copy "&local=true" every time a video failes to load...

**Comment by Perflyst** [comment]
Enable "Proxy videos?" in settings (works without account) or install an addon like [Invidition](https://addons.mozilla.org/en-US/firefox/addon/invidition)

**Comment by CuloArdido** [comment]
Add three other videos to the table and it is also [this](https://invidio.us/watch?v=5T3ypBvd3YY) that apparently loads well but it is not possible to play it (not even with local=true) and the allowedRegions field is empty. When trying to watch it on YouTube, the following message appears:
This video contains content from WMG, who has blocked it in your country on copyright grounds.

**Comment by omarroth** [comment]
> I understand that videos blocked in my country or in the United States cannot be viewed directly, but what about those that are accessible in both regions? Why can't they be viewed unless a proxy is used?

It might make sense to imagine each request to YouTube as having an associated region, so for example:
- Invidious requests info for video `X` (from region `US`)
- YouTube provides video files (with associated region `US`)
- Client requests file (from region `DE`)

At this point YouTube may block access if `DE` is geo-blocked, **or** (occasionally) if it doesn't match the original request region (`US`). It's currently not possible to distinguish between the two since YouTube will return a generic 403 for both cases (see below).

> instead could show a more explanatory message like the one I proposed in my previous comment (assuming that it can unequivocally detect that the video can not be loaded due to a geographical block and not for other reasons)

I don't think this is possible currently. When the client attempts to load a video, YouTube will return 403 for several reasons:
- URL can't be accessed from the user's region
- URL doesn't match original region where info was requested.
- URL is expired
- URL is malformed (invalid signature, etc.)
- ...

> in case the country where Invidious is running is blocked, then proxify the video automatically without having to add local=true in the url

I'm assuming what you mean here is to automatically proxy videos if Invidious thinks the client won't be able to view it. The issue is that YouTube will sometimes (not always or consistently) block access for clients that aren't in the same region as the server.

My main issue with adding `&local=true` when encountering `The media could not be loaded...` is because AFAIK it is currently not possible to identify the exact cause of the error. Since proxying puts a larger load on instance, I'd prefer it to be a choice made explicitly by the user.

**Comment by tuxayo** [comment]
Can someone confirm if the following is a duplicate of this issue.

https://invidious.13ad.de/watch?v=5yNlPL9unj0&local=true

https://invidio.us/watch?v=5yNlPL9unj0&hl=en-US&local=true

So, still doesn't work when using proxy from a german or US instance, is that another issue?

**Comment by Perflyst** [comment]
Yes, I can reproduce this on the two instances. Also in Finland on fi.invidious.snopyta.org

```
HTTP load failed with status 500. Load of media resource https://invidio.us/latest_version?id=5yNlPL9unj0&itag=22&local=true failed. watch
VIDEOJS: ERROR: (CODE:4 MEDIA_ERR_SRC_NOT_SUPPORTED) The media could not be loaded, either because the server or network failed or because the format is not supported. 
Object { code: 4, message: "The media could not be loaded, either because the server or network failed or because the format is not supported." }
```

Without proxy it results in 403 instead of 500

server log
<details>
2019-08-06 19:29:56 UTC 403 GET /api/manifest/hls_playlist/expire/1565066026/ei/yq5IXf6qEdPOyQXU_JKIAg/ip/95.216.24.230/id/czizLiRZ988.0/itag/94/source/yt_live_broadcast/requiressl/yes/ratebypass/yes/live/1/goi/160/sgoap/gir%3Dyes%3Bitag%3D140/sgovp/gir%3Dyes%3Bitag%3D135/hls_chunk_host/r4---sn-axq7sn7e.c.youtube.com/playlist_type/DVR/mm/30/mn/sn-axq7sn7e/ms/nxu/mv/u/mvi/3/nh/IgpwcjAzLmxlZDAzKgkxMjcuMC4wLjE/pl/21/sc/yes/dover/11/keepalive/yes/mt/1565043557/sparams/expire,ei,ip,id,itag,source,requiressl,ratebypass,live,goi,sgoap,sgovp,playlist_type/sig/ALgxI2wwRgIhAI5iFdy6Cb0EO7qmFqYBLLOzpxhvOUg_2X38jtjKs6jkAiEAgtWQWbzlmS1JxD3bVkEZWMQSebc1tK37J9aI6vZZjkY%3D/lsparams/hls_chunk_host,mm,mn,ms,mv,mvi,nh,pl,sc/lsig/AHylml4wRgIhAPp6-IzuPid9959PqxEVkgUnYs1REl-0MsZXJLakNte5AiEAz9Izkqhrd27Md4jGHUskw2UYiXNeehZZr3a7OOqb2y0%3D/playlist/index.m3u8?local=true 80.24ms
2019-08-06 19:29:12 UTC 200 POST /api/v1/auth/notifications?fields=videoId,title,author,authorId,publishedText,published,authorThumbnails,liveNow 44004.58ms
2019-08-06 19:29:55 UTC 200 GET /api/v1/comments/5yNlPL9unj0?format=html&hl=en-US&thin_mode=false 953.26ms
2019-08-06 19:29:55 UTC 200 GET /vi/5yNlPL9unj0/maxres.jpg 1198.3ms
2019-08-06 19:29:04 UTC 200 POST /api/v1/auth/notifications?fields=videoId,title,author,authorId,publishedText,published,authorThumbnails,liveNow 52013.74ms
2019-08-06 19:29:57 UTC 200 GET /api/v1/auth/subscriptions?fields=authorId 3.68ms
2019-08-06 19:29:11 UTC 200 POST /api/v1/auth/notifications?fields=videoId,title,author,authorId,publishedText,published,authorThumbnails,liveNow 46049.84ms
2019-08-06 19:29:58 UTC 200 GET /api/v1/auth/subscriptions?fields=authorId 3.56ms
2019-08-06 19:29:58 UTC 403 GET /api/manifest/hls_playlist/expire/1565024837/ei/5Q1IXceEB4rb1wLV-5lA/ip/2a01:4f8:c0c:782::2/id/coYw-eVU0Ks.0/itag/96/source/yt_live_broadcast/requiressl/yes/ratebypass/yes/live/1/goi/160/sgoap/gir%3Dyes%3Bitag%3D140/sgovp/gir%3Dyes%3Bitag%3D137/hls_chunk_host/r6---sn-4g5e6nez.c.youtube.com/playlist_duration/30/manifest_duration/30/gcr/de/playlist_type/DVR/mm/30/mn/sn-4g5e6nez/ms/nxu/mv/u/mvi/5/pl/40/sc/yes/dover/11/keepalive/yes/mt/1565002628/sparams/expire,ei,ip,id,itag,source,requiressl,ratebypass,live,goi,sgoap,sgovp,playlist_duration,manifest_duration,gcr,playlist_type/sig/ALgxI2wwRQIgNIAU4xnPida4xfqFWjb7zewKwsU-uoXOJXfNtnWg4YwCIQC4NchvpMpMPUFnVnMiph424yEg7H042eEId3Tf2OsP-A%3D%3D/lsparams/hls_chunk_host,mm,mn,ms,mv,mvi,pl,sc/lsig/AHylml4wRQIgU7lldXTORwfvY0f3m6P5gePzmOXSt7pnyORU6r8W1AQCIQDAsOXcPQl_-7Z-fKYsF1eXj0cSWPfjSWNf1OEDfwtWfw%3D%3D/playlist/index.m3u8?local=true 45.34ms
</details>

**Comment by gloschtla** [comment]
I am having this same error 

> The media could not be loaded, either because the server or network failed or because the format is not supported.

Does it have [any relation with my browser](https://github.com/omarroth/invidious/issues/931)?

**Comment by nathgit** [comment]
> Is there a way I can get invidious to add "&local=true" to every URL by default? This fixes almost every unplayable video for me. But it's a pain to copy "&local=true" every time a video failes to load...

you could probably do that with a greasemonkey script. I don't know how to write one, but I'm sure you could write a script to append a snippet of text to a url automatically. I know you can do with a bookmarklet, but you have to manually run that bookmarklet.

**Comment by nathgit** [comment]
> Please add &local=true at the end and it should work
This didn't work for me. The url is 
https://invidio.us/watch?v=Jkx4BMLN4kw. I also tried the invidious.snopyta.org with the same results. I even tried using '&hl=en-US&local=true', with no change.

**Comment by nathgit** [comment]
I just tried the link again & it played this time adding '&hl=en-US&local=true' for the invidio.us link only. It still didn't work with the snopyta link. If this isn't going to play videos reliably I may have to find another alternative when invidio.us shuts down, because this is getting really annoying.

**Comment by nathgit** [comment]
Is there still no solution? adding `&local=true` & `&hl=en-US&local=true` doesn't do anything. It just seems to happen randomly, but it happens more & more.

**Comment by nathgit** [comment]
Does anybody know any good alternatives to this? I'm fed up with it. between this problem & it occasionally refusing to load search results it's getting to be a pain.

**Comment by fliegentoeter** [comment]
> Does anybody know any good alternatives to this? I'm fed up with it. between this problem & it occasionally refusing to load search results it's getting to be a pain.

I switched to Freetube on my desktop and Newpipe on my mobile phone. None of them are perfect, but get updated rather quickly when they are broken...

**Comment by SamantazFox** [comment]
I guess that it has to do with copyrights. Many of these videos can't even be played as a youtube `embed`. We'll have to investigate, there are many things that have to be rewritten in order to mimic the youtube player's behavior and maybe bypass that.

We also know that the decryption function job (who reverse engineers the video protection) is broken.

**Comment by SamantazFox** [comment]
Just to be sure, I tried them all. Here are the results:

| Video ID    | Type of restriction                    |
+-------------+----------------------------------------+
| Yc5sgGMmGUQ | DRM-protected (level 1)                |
| INS2Dsm1_Ew | geo-blocked                            |
| h4bX2boNO5g | geo-blocked                            |
| iOSUriAFLgk | blocked because of copyright violation |
| vuhHw2VeGCY | blocked because of copyright violation |
| d8Qd4K8Medc | DRM-protected (level 1)                |
| FxhJLCRsz98 | DRM-protected (level 1)                |
| I1OOIgcEADk | DRM-protected (level 2)                |
| 0fo-8LkzC9U | DRM-protected (level 2)                |
| 5Mcj_YtQhbQ | DRM-protected (level 2)                |
| 5-3KI7up4eM | Account terminated                     |
| kMML6qvVT54 | DRM-protected (level 1)                |
| ccq8ctJ4PKY | DRM-protected (level 1)                |
| SI4Wf_qse5k | DRM-protected (level 1)                |
| dHVBxb_sg_k | DRM-protected (level 1)                |
| WXToe26mwTo | DRM-protected (level 1)                |
| wmmEvPqN4Zk | blocked because of copyright violation |
| KMVfkku-B8o | Account terminated                     |
| c26IXiQo5w0 | Account terminated                     |
| cpzL0H6mkQc | Account terminated                     |
| mJqS3R8dfFM | Account terminated                     |
| _xMucDVHag8 | Account terminated                     |
| Q3FBADNgPS0 | Account terminated                     |
| yYo1jsM-KhA | removed because of copyright violation |
| s1S--jFuly4 | DRM-protected (level 2)                |
| vuK1U76dRNE | Unavailable (no reason)                |
| P7l6iQe0x2Q | DRM-protected (level 1)                |

DRM protection:
 - level 1: plays on youtube.com/embed/
 - level 2: plays only on youtube main site

**Comment by 89z** [comment]
https://github.com/iv-org/invidious/issues/2354

**Comment by Zero3K** [comment]
What about z0DRRMI7-Sw?

**Comment by SamantazFox** [comment]
> What about z0DRRMI7-Sw?

This one is not available in Japan apparently.
Plays fine from an invidious instance hosted in Germany.

**Comment by unixfox** [comment]
Closing in favor of https://github.com/iv-org/invidious/issues/602
